### DISCUSSION TITLE
i18n locale don't translate every label.

### DISCUSSION THREAD

**Discussion by Ostrowin** [GrapesJS/grapesjs#4115] [Q&A]
grapesjs version: 0.18.1

I would like to translate editor, so what i've done is :
- import lang locale
- on init add i18n locale
![i18n](https://user-images.githubusercontent.com/35689589/151942750-27a9bbcf-778d-4c5a-9df8-c6b4e3ee83b9.png)

Not every translation appeared, for example and the most important, StyleManager sector labels are not translated:
![smLabels](https://user-images.githubusercontent.com/35689589/151943513-fa1ac062-2bbd-4695-8613-3cbca73ddd10.png)

or StyleManager empty:
![smempty](https://user-images.githubusercontent.com/35689589/151943923-0db366de-0c4a-4574-b659-5f974aea9900.png)

Trait empty:
![smempty2](https://user-images.githubusercontent.com/35689589/151944222-afef9566-2b8a-4dc0-b4ba-bd97b187e441.png)

or Style Manager Classes and Selected words:
![smClassesSelected](https://user-images.githubusercontent.com/35689589/151944260-6762cdbe-0907-49bb-8138-b4a197d0955d.png)

Rest, especially style manager css properties are translated fine.
Does anyone know how to translate (or change labels) that examples? Especially style manager sectors labels?

Have a good day.

**Comment by artf** [comment]
You can submit a PR for this file https://github.com/artf/grapesjs/blob/dev/src/i18n/locale/pl.js
The base reference file is this one https://github.com/artf/grapesjs/blob/dev/src/i18n/locale/en.js
